charts/k8tz/values.yaml

```diff
@@ -31,6 +31,10 @@ webhook:
 
   caBundle: |
 
+  tlsMinVersion: ""
```

Comment:
(nit) could you please move those values to line 18 and remove the extra line between them? thanks.

Reply:
done